GitUI/Editor/FileViewer.cs

```diff
@@ -1027,7 +1027,7 @@ private Task ViewItemAsync(string fileName, bool isSubmodule, Func<Image?> getIm
                                     ResetView(ViewMode.Text, null);
 
                                     var text = getFileText();
-                                    var summary = new StringBuilder()
+                                    StringBuilder summary = new StringBuilder()
```

Comment:
Intentional?

Reply:
Also, was there not a Roslyn codefix to make these changes? If one exists, why didn't you use it? If it has issues, I'd like to report them.

Reply:
This was intentional. new() cannot be used here and I reverted the change. In this case I followed IDE0008 and kept the type. Will remove that from this PR.

It is not possible to apply IDE0090 without removing `var` first. So IDE0008 has to be applied which removes all var declarations before IDE0090 can be applied.
Adding a commit to apply IDE0090, there are some usages, especially in definitions.
Who could imagine a compiler was better than a regex?